elements/pf-dropdown/pf-dropdown-group.ts

```diff
@@ -15,17 +16,60 @@ export class PfDropdownGroup extends LitElement {
 
   static override readonly shadowRootOptions: ShadowRootInit = { ...LitElement.shadowRootOptions, delegatesFocus: true };
 
+  /**
+   * Disable the dropdown
+   */
+  @property({ type: Boolean, reflect: true }) disabled = false;
+
   /**
    * The label for the group of dropdown items.
    */
   @property({ reflect: true }) label?: string;
 
+
+  #internals = InternalsController.of(this, { });
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.addEventListener('focusin', this.#onMenuitemFocusin);
+    this.addEventListener('click', this.#onMenuitemClick);
+  }
+
+  protected updated(changed: PropertyValues<this>): void {
+    if (changed.has('disabled')) {
+      this.#internals.ariaDisabled = `${!!this.disabled}`;
+    }
+  }
+
   render() {
     return html`
       <p ?hidden="${!this.label}" role="presentation">${this.label}</p>
       <slot></slot>
     `;
   }
+
+  /**
+   * handles focusing on an option:
+   * updates roving tabindex and active descendant
+   */
+  #onMenuitemFocusin(event: FocusEvent) {
+    if (this.disabled) {
+      event.preventDefault();
+      event.stopPropagation();
```

Comment:
do we need to stop propagation? is preventDefault enough?

I think we should also leave a "why comment" here and below, to prevent confusion in the future, since the reader may not immediately grok that disabled dropdowns are still permitted to open